### DISCUSSION TITLE
Playbooks - Edit via GUI then Reverts back

### DISCUSSION THREAD

**Discussion by Git-Me-Some-Hub** [Security-Onion-Solutions/securityonion#6383] [Unsupported Versions] [answered]
2.3.80 - ISO - Master/Search
Play: Active Directory Replication from Non Machine Account

Hello, 

I could not be understanding the proper use of Playbook plays, but I feel like I went back through the documentation and did not see this specifically called out. I do see that you cannot really change the default Community Playbooks as those will sync back up with Github versions, so I copied it over to a brand new playbook:599. I then tweaked it by adding the valid accounts that were making noise in the alerts. 

Now I am getting alerts on both the new play I tweaked and the old one it seems to have reverted back. Both reference the same yaml file, and play number. 

My question is, am I not suppose to tweak the play after I initially created them, or what? How do I fix this?

Alert SOC: 
![Playbook01](https://user-images.githubusercontent.com/17930434/143931334-a248a0c9-baf4-4064-9077-e77b2a5103ce.PNG)

Old: 
![Playbook_599_old](https://user-images.githubusercontent.com/17930434/143931425-23de7fa2-c9f4-4a7d-8979-7e06efcdeec4.PNG)

New: 
![Playbook_599_new](https://user-images.githubusercontent.com/17930434/143931442-dd64c1c1-97eb-4717-8311-00ffc923c49d.PNG)

**Comment by defensivedepth** [comment, marked as answer]
Yes, you will need to duplicate the community plays, in order to make changes to them (there will be a better process for this in the future).

Duplicate it by creating a New Play and pasting in the Sigma from the community play. That will generate a new PlayID, which will make it so that it doesn't get updated on a daily basis.

**Reply by Git-Me-Some-Hub** [reply]
Sorry but this does not answer my question, I did read that you need to duplicate the community playbooks, which I did. However, I copied that over, and then modified it... That is where I am sitting right now. I modified the already copied community playbook and it is not sticking. 

Does that make sense?

**Reply by defensivedepth** [reply]
What process did you use to duplicate it?

**Reply by Git-Me-Some-Hub** [reply]
Just a copy of the sigma rule and pasted it in to create a new rule. looking at the new one I created I guess I do see the Rule ID the exact same, I also might have had the name the exact same at the time too. Does that matter?

**Reply by defensivedepth** [reply]
The ID is what matters. Gen one from here: https://www.uuidgenerator.net/version4 and use that one. That should fix the problem.

**Reply by Git-Me-Some-Hub** [reply]
Perfect, Most the other times I created new one I did not put in an ID as I really did not know where to generate one, or if it was needed. 

I did try and add a new ID to the ones I already copied, but it did not seem to stick still. I have deactivated those as well and create yet another new one. I will remember this going forward. Do you think it would be helpful to add a bit more of this to the Documentation?

**Reply by defensivedepth** [reply]
Ya, good call. Will get the docs updated to clarify that.
